xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-bootswatch/src/main/resources/FlamingoThemes/Superhero.xml

```diff
@@ -2125,7 +2138,7 @@ div.main {
   box-shadow: none;
 }
 
-@xwiki-panel-header-text: @navbar-default-color;
+@panel-header-text: @navbar-default-color;
```

Comment:
Should this either a) set the corresponding theme variable or b) also set the fallback value?

Reply:
For better backwards compatibility, and consistency with the choice we made in `variablesInit.vm`, I added back the former value as a fallback in https://github.com/xwiki/xwiki-platform/pull/2497/commits/afc21a6f696fc2ef84bb681067afe1ad954535b2 :+1: